packages/aws-cdk/test/api/bootstrap.test.ts

```diff
@@ -1,68 +1,68 @@
-/* eslint-disable import/order */
-import { CreateChangeSetInput } from 'aws-sdk/clients/cloudformation';
+import {
+  CreateChangeSetCommand,
+  CreateStackCommand,
+  DeleteStackCommand,
+  DescribeChangeSetCommand,
+  DescribeStacksCommand,
+  ExecuteChangeSetCommand,
+  GetTemplateCommand,
+  StackStatus,
+  UpdateTerminationProtectionCommand,
+} from '@aws-sdk/client-cloudformation';
 import { parse } from 'yaml';
 import { Bootstrapper } from '../../lib/api/bootstrap';
-import { deserializeStructure } from '../../lib/serialize';
-import { MockSdkProvider, SyncHandlerSubsetOf } from '../util/mock-sdk';
+import { legacyBootstrapTemplate } from '../../lib/api/bootstrap/legacy-template';
+import { deserializeStructure, serializeStructure, toYAML } from '../../lib/serialize';
+import { MockSdkProvider, mockCloudFormationClient, restoreSdkMocksToDefault } from '../util/mock-sdk';
 
 const env = {
   account: '123456789012',
   region: 'us-east-1',
   name: 'mock',
 };
 
+const templateBody = toYAML(deserializeStructure(serializeStructure(legacyBootstrapTemplate({}), true)));
```

Comment:
What's the difference between this and:

```
const templateBody = serializeStructure(legacyBootstrapTemplate({}), false);
```
?

Reply:
This is silly looking but unfortunately necessary. If we only serialize it, the template body isn't formatted correctly due to the escape characters that get added in. If we skip the deserialization/serialization part and just do toYAML, we get the same output.